CHANGELOG.md

```diff
@@ -1,7 +1,31 @@
+# 4.0.0 (2020-05-22)
+
+### New features
+
+- Updated the pipeline to transpile to the [Angular Package Format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview). [#16](https://github.com/blackbaud/skyux-link-records/pull/16)
```

Comment:
Seems a little weird to me that this points to an "Initial commit" PR instead of an "Angular 9" PR that the other changelogs point to. I assume this is OK based on looking through the file changes in the PR, but just making sure.

Reply:
Good observation. I just misnamed that PR, but it should be the same as the others.